### PR TITLE
Markdown headings, typos and lead-in for additional identities

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

File: trust-store-trust-policy-specification.md [hunk 1]
```diff
@@ -11,14 +11,16 @@ The document consists of the following sections:
 Users who consume and execute the signed artifact from a registry need a mechanism to specify the trusted producers.
 This is where Trust Store is used.
```
Comment (gokarnm) [inline trust-store-trust-policy-specification.md:12]: We should start by introducing trust policy, and then talk about trust stores. Trust stores by themselves does not configure trusted producers.

File: trust-store-trust-policy-specification.md [hunk 2]
```diff
@@ -11,14 +11,16 @@ The document consists of the following sections:
...
 
-Trust store allows users to specify two kinds of identities:
+Trust store allows users to specify two kinds of identities, with additional identities coming in future releases:
 
 - **Certificates**: These are the signing certificates or certificates in the certificate chain of the signing certificate.
```
Comment (gokarnm) [inline trust-store-trust-policy-specification.md:16]: These are typically root certificates of of issuing authorities (public CA, internal CAs) that the user trusts, and which will issue the signing certificates.

File: trust-store-trust-policy-specification.md [hunk 3]
```diff
@@ -100,17 +102,19 @@ Since revocation check requires network call and network call can fail because o
...
     "version": "1.0",
     "trustPolicies": [
         {
             "name": "verify-signature",
             "scopes": [
-                "wabbit-networks.io/software/product1"
-                "wabbit-networks.io/software/product2" ],
+                "registry.wabbit-networks.io/software/net-monitor"
+                "registry.wabbit-networks.io/software/net-logger" ],
             "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
```
Comment (gokarnm) [inline trust-store-trust-policy-specification.md:118]: Lets update the trust store names to correspond to the example. Need not demonstrate it referencing two trust stores, more confusing for the reader.

File: trust-store-trust-policy-specification.md [hunk 4]
```diff
@@ -180,7 +184,7 @@ Property descriptions
     - **`timestampRevocation`**(*string*): This REQUIRED property specifies whether implementation should check for timestamping certificate and certificate-chain revocation status or not and what implementation must do if this revocation check fails.
       Supported values are `enforceWithFailOpen`, `enforceWithFailClose`, `warn` and `skip`.
 
-Value descriptions
+#### Value descriptions
```
Comment (gokarnm) [inline trust-store-trust-policy-specification.md:187]: We need to take a pass over the config options and suggest better ones, failopen/failclose are confusing to most users.

**Review by gokarnm — CHANGES_REQUESTED** [review]

File: trust-store-trust-policy-specification.md (hunk 3 above)
Comment (priteshbandi) [reply trust-store-trust-policy-specification.md:118]: Why are we adding `registry` as prefix? Is it just part of registry URI?

Comment (SteveLasker) [reply trust-store-trust-policy-specification.md:118]: > Why are we adding registry as prefix? Is it just part of registry URI?

Yeah, it's just the registry name. The presumption is wabbit-networks.io is their public website :)

Comment (SteveLasker) [reply trust-store-trust-policy-specification.md:118]: > Lets update the trust store names to correspond to the example. Need not demonstrate it referencing two trust stores, more confusing for the reader.

@gokarnm, are you suggesting I sync the trust stores in line 118?

File: trust-store-trust-policy-specification.md (hunk 4 above)
Comment (SteveLasker) [reply trust-store-trust-policy-specification.md:187]: Agreed. Can we open an issue? There's no change here, so just want to keep the PRs separate.

**Comment by SteveLasker** [comment]
@gokarnm, can we queue the additional changes to another PR? This was just a set of typos and heading fixes.

**Review by gokarnm — APPROVED** [review]

**Pull request merged**